source/upgrade/notify-admin.rst

```diff
@@ -0,0 +1,25 @@
+Notify admin
```

Comment:
@justinegeffen The ``EnableAPITriggerAdminNotifications`` config setting may also need to be documented here:
 - https://docs.mattermost.com/manage/telemetry.html
 - https://docs.mattermost.com/configure/configuration-settings.html

Reply:
Thank you! I didn't even think of that. :)

Reply:
Configuration: https://github.com/mattermost/mattermost-server/pull/20777/files#diff-07146319e6e0a169b23444e4db03e086b60c72f16996920a8eabfb057841c8ee

Telemetry: https://github.com/mattermost/mattermost-server/pull/20777/files#diff-7bc66d203ca22000f3fa14a2f1db4d3f013baf12a3b7930f4ad52d1f82b135bb

Reply:
Thanks so much @amyblais!